src/data-mgt/python/jobs/device_registry.py

```diff
@@ -0,0 +1,131 @@
+import json
+import os
+import requests
+
+AIRQO_API_BASE_URL = os.getenv("AIRQO_API_BASE_URL")
+
+MEASUREMENT_UNITS = dict({
+    "temperature": "°C",
+    "relHumid": "%",
+    "pm10ConcMass": "μg/m3",
+    "pm10ConcNum": "micrograms",
+    "pm1ConcMass": "μg/m3",
+    "pm1ConcNum": "micrograms",
+    "pm2_5ConcMass": "μg/m3",
+    "pm2_5ConcNum": "micrograms",
+    "no2Conc": "ppb",
+    "vocConc": "ppb",
+    "co2Conc": "ppb",
+})
+
+
+def single_component_insertion(data, tenant):
```

Comment:
This method/function might need to be renamed since we are no longer doing component insertions.